tests/foreman/ui/test_role.py

```diff
@@ -240,6 +242,27 @@ def test_positive_delete_cloned(self):
             self.role.delete(new_name)
```

Comment:
nope, we automatically verify that entity is deleted in the framework code itself. We did that to avoid code and execution time redundancy.
`self.assertIsNone(self.role.search(new_name))` will take unnecessary 15 seconds to execute. Imagine if we keep all asserts like that in the code